src/NetworkPort.php

```diff
@@ -1798,13 +1738,14 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
             }
             $nbAggregates = countElementsInTable(
                 'glpi_networkportaggregates',
+                //FIXME This seems prone to false positives. Assume that networkports_id_list is a JSON array. Searching %1% would match 1, 10, 31, 100, etc.
```

Comment:
Values are surrounded by quotes, so searching for 1 will be done using `%"1"%` and would not match `"10"` nor `"31"`.
```suggestion
```